src/git.js

```diff
@@ -30,6 +31,28 @@ module.exports.refExists = ref => {
   return result.status === 0;
 };
 
+/* Returns the SHA-1 hash of the latest commit in the given ref. */
+module.exports.shaForRef = ref => {
```

Comment:
FYI most of these are available as env variables in GitHub Actions runners: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

But if this approach appears to work just fine 🤷🏽

Reply:
I was able to remove `shaForRef()` but the commit message isn't in the environment or the context object, so I kept that one in here.